recipes/opentelemetry-distro/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "opentelemetry-distro" %}
+{% set version = "0.36b0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/opentelemetry_distro-{{ version }}.tar.gz
+  sha256: df8cb0b4428079c2391681504772ae4e79580ffecec4470b49ff9f28e1490304
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+    - hatchling
+  run:
+    - python >=3.7
+    - opentelemetry-api >=1.12,<2.dev0
+    - opentelemetry-instrumentation ==0.36b0
+    - opentelemetry-sdk >=1.13,<2.dev0
+
+test:
+  imports:
+    - opentelemetry_distro
```

Comment:
It's complaining that `import opentelemetry_distro`. What's the right way to test (using Python) that this package is properly installed?